### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by Gh-Reza**
## Changes:

- Configured the store
- Added the booksSlice
- Added the catagoriesSlice
- Updated the readme

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @Gh-Reza,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the live demo link from your readme header, since it has no section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/catagories/catagoriesSlice.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const catagoriesSlice = createSlice({
+  name: 'catagory',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state,
```
Comment (DammyShittu) [inline src/redux/catagories/catagoriesSlice.js]: - [ ] The category reducer should always return the text "Under Construction". Kindly refactor to that. See this example:

  ```
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction" ? "Under construction" : state.categories
    },
  ```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Gh-Reza,

## Highlights
✔️ Redux toolkit for store

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Gh-Reza👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**